tests/jballerina-unit-test/src/test/resources/test-src/expressions/conversion/native-conversion.bal

```diff
@@ -855,14 +857,22 @@ public class O1 {
   public int y = 0;
 }
 
-function testTupleConversion1() returns [T1, T1]|error {
+function testTupleConversion1() {
     T1 a = {};
     T2 b = {};
-    [T1, T2] x = [a, b];
+    [T1, T2] x1 = [a, b];
```

Comment:
Shall we use the already defined types `T1_T1` & `T1_T2` here and similar cases?
We can re-use the type.

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/31779/commits/93a63a8b643f9e14f56219f8cc2aba8a54280939